README.md

```diff
@@ -15,4 +15,5 @@ Development
 -----------
 - [Issue tracker](https://bugs.alliedmods.net): Issues that require back and forth communication
 - [Building AMXX](https://wiki.alliedmods.net/Building_AMX_Mod_X): Instructions on how to build AMXX itself using [AMBuild](https://github.com/alliedmodders/ambuild)
+- [AMX Mod X API](http://www.amxmodx.org/api/): AMX Mod X API reference generated from include files
```

Comment:
`https` please

Reply:
Hmm, ok but amxmodx.org sending me to http://www.amxmodx.org/api, fix server config please :p

Reply:
Not sure what you mean, the website is fully in https and the links as well?

Reply:
When you typing 'amxmodx.org' in browser(without http or https), it sending you to http version with www

Reply:
I don't have this behavior, but it might be the browser cache.
Anyway, https should be enforced by default even if the website allows without.